scenario_09_sandbox.lua

```diff
@@ -180,7 +180,7 @@ function setConstants()
 	revert_timer = revert_timer_interval
 	plotRevert = revertWait
 	tractor_sound = "sfx/emp_explosion.wav"
-	tractor_sound_power = .25
+	tractor_sound_power = .05
```

Comment:
just FYI unless things have changed (very possible) sadly sound_power doesnt quite work as desired - there is randomisation and IIRC the volume is abs(sound_power)
I guess this should be checked its still an issue and formalised into a EE bug report
likewise the lack of a way to create a BeamEffect without playing a sound probably should be a bug report (though was the crash on changing tractor sound to invalid fixed?)

Reply:
There are buttons to adjust the power on the fly in game:
+Custom > +Snippet > +Tractor Sound > then there are various choices for raising and lowering the sound power. You can also change the default sound. The most benign sound is the Button sound.

I put in those controls to get a feel for what's best in terms of sound, then I'd adjust the defaults to whatever experimentation revealed.

Reply:
I'll revert this change, no longer needed.